docs/building-apps/moneygram-access-integration-guide.mdx

```diff
@@ -358,6 +517,53 @@ Note that this code does not handle [path payments] or [claimable balances], two
 
 <CodeExample>
 
+```ts
+// The Wallet SDK does not support payments streaming yet so let's build
```

Comment:
IMO it's a good functionality for wallet SDK to have 🤔

Reply:
yep

we have this task on the backlog: https://stellarorg.atlassian.net/jira/software/c/projects/WAL/boards/37/backlog?selectedIssue=WAL-1444